tools/setup_minio.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+## see https://github.com/minio/minio/issues/4769 for more examples
+  
+minio_docker_name="mongooseim-minio"
+minio_access_key="AKIAIAOAONIULXQGMOUA"
+minio_secret_key="CG5fGqG0/n6NCPJ10FylpdgRnuV52j8IZvU7BSj8"
+minio_bucket="mybucket"
+
+docker rm -v -f "${minio_docker_name}" || echo "Skip removing previous container"
+
+docker run -d -p 9000:9000 \
+    --name "${minio_docker_name}" \
+    -e "MINIO_ACCESS_KEY=${minio_access_key}" \
+    -e "MINIO_SECRET_KEY=${minio_secret_key}" \
+    minio/minio server /data
+
+mc_cmd="$(cat <<-EOF
+  while ! nc -z minio 9000; do echo 'Wait minio to startup...' && sleep 5; done;
+  mc config host add myminio http://minio:9000 "${minio_access_key}" "${minio_secret_key}" && 
+  mc mb "myminio/${minio_bucket}" &&
+  mc policy set download "myminio/${minio_bucket}"
+  exit 0
+EOF
+)"
+
+docker run --rm --entrypoint sh \
+    --link "${minio_docker_name}:minio" \
+    minio/mc -c "${mc_cmd}"
```

Comment:
EOL missing in the last line, some editors like VSC tend to do that, but that is not POSIX compliant:
https://stackoverflow.com/a/729795/8853275

Reply:
FYI, POSIX doesn't enforce a new line at the end of the file. it even declares an [incomplete line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_195)

Reply:
Yeah, but it is, precisely, an *_in_*complete line 🤷‍♂ Please fix it, most of the POSIX tools don't like it. Including github showing a 🚫 symbol at the end of the file.

Reply:
My comment was about being "POSIX compliant". but as you mention POSIX tools, could you be more specific? which exactly POSIX tools? please don't say `wc` and `cat`. i don't belive that counting line numbers and  text files concatenation are so critical. and github shows that symbol in the end of the file, wherever it is.